### PR TITLE
Minor fixes to the text

### PR DESCRIPTION
**Pull request by jmenglund**

### CODE REVIEW

**Review by shreyasshivakumara — APPROVED** [review]

File: _includes/data-access-table.html [hunk 1]
```diff
@@ -54,16 +54,16 @@
     <div class="arrow arrow-down"></div>
   </div> 
   <div>
-    Identify the dataset(s) you would like to get access to.
+    Identify the dataset(s) you want to request access for.
```
Comment (norling) [inline _includes/data-access-table.html:57]: Access it "to" stuff.
```suggestion
    Identify the dataset(s) you want to request access to.
```

File: _includes/data-access-table.html [hunk 2]
```diff
@@ -54,16 +54,16 @@
...
 
-  <div class="steps">2. Apply for access
+  <div class="steps">2. Request access
     <div class="arrow arrow-down"></div>
   </div> 
   <div>
     Identify the Data Access Committee (DAC) affiliated with the dataset(s)
-    you are interested in. Then, contact the DAC to start your data access
-    application.
+    you wish to request access for. Then, contact the DAC to start your data
```
Comment (norling) [inline _includes/data-access-table.html:65]:
```suggestion
    you wish to request access to. Then, contact the DAC to start your data
```

File: _includes/quick-guide-table.html [hunk 3]
```diff
@@ -54,26 +54,26 @@
     <div class="arrow arrow-down"></div>
   </div> 
   <div>
-    Assess whether SDA is the right place for your data. The SDA can only take
-    certain types of data and the data submitted to the repository must require
-    controlled access (i.e. that formal applications must be made to
-    access submitted files). Read about how to <a href="../is-sda-suitable.html">find
-    out if SDA is suitable or not</a>.
+    Assess whether SDA is the right place for your data. The repository is for
```
Comment (norling) [inline _includes/quick-guide-table.html:57]: This should probably be "the SDA" right?
```suggestion
    Assess whether the SDA is the right place for your data. The repository is for
```

**Review by dbampalikis — APPROVED** [review]

File: data-access.md [hunk 4]
```diff
@@ -7,8 +7,8 @@ titles:
 key: page-single
 ---
 
-This page describes how you can get access to datasets that has been submitted
-to SDA. For any questions regarding data access, please contact the 
+This page describes how you request access to datasets deposited in SDA. For
```
Comment (norling) [inline data-access.md:10]:
```suggestion
This page describes how you request access to datasets deposited in the SDA. For
```

File: _includes/quick-guide-table.html [hunk 5]
```diff
@@ -54,26 +54,26 @@
...
 
   <div class="steps">2. Create a submission request
     <div class="arrow arrow-down"></div>
   </div>
   <div>
-    The SDA is under development and only able to handle a limited number of
-    submission requests. If you think that SDA is the right place for your data
-    and want to be added to our waiting list, please fill in the 
+    The SDA is under development and can currently only handle a limited number
+    of submission requests. If you consider SDA to be the right place for your
```
Comment (norling) [inline _includes/quick-guide-table.html:69]:
```suggestion
    of submission requests. If you consider the SDA to be the right place for your
```

**Review by norling — APPROVED** [review]
Some nitpicky grammar stuff, but otherwise it looks great! :)

**Pull request merged**